src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+import Header from "./components/Header";
+import MainPage from "./components/MainPage";
+import Footer from "./components/Footer";
+
+function App() {
+  let [data, setData] = useState(null);
+
+  let [lang, setLang] = useState("fr");
+
+  let [frFlagWidth, setFrFlag] = useState("100%");
+  let [enFlagWidth, setEnFlag] = useState("50%");
+
+
+  let mainPageHeight = (data === null) ? "84%" : "fit-content";
```

Comment:
pq pas definir max-height ?

Reply:
ou juste utilise flex-grow?

Reply:
Un lien utile: https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flexbox-properties

Reply:
C'est bon je l'ai fait avec min-height dans le css du footer et de mainpage